### PR TITLE
Adds option to disable ssl verification and use self signed certificates

### PR DESCRIPTION
**Pull request by russmac**
These changes allowed me to use a self signed cert.

### CODE REVIEW

File: confidant_client/__init__.py [hunk 1]
```diff
@@ -87,6 +88,7 @@ def __init__(
             configuration from. First file found will be used. Default:
                 ['~/.confidant', '/etc/confidant/config']
             profile: profile to read config values from.
+            verify:  Whether we verify the server’s TLS certificate.
```
Comment (ryan-lane) [inline confidant_client/__init__.py]: It looks like the ' here is unicode, which is causing tests to fail.

**Comment by ryan-lane** [comment]
+1 after tests

File: confidant_client/__init__.py (hunk 1 above)
Comment (russmac) [reply confidant_client/__init__.py]: Thanks for spotting it, Looks like my copy paste from the official (requests) documentation. Ill fix and update PR.

**Comment by ryan-lane** [comment]
Thanks for the PR!

**Pull request merged**